log.go

```diff
@@ -389,10 +389,11 @@ func (l *raftLog) term(i uint64) (uint64, error) {
 	}
 
 	// The valid term range is [index of dummy entry, last index].
```

Comment:
We no longer have a "dummy entry" variable, so it's not clear what this comment is referring to. FWIW, I never find it clear when reading this code why `i == l.firstIndex() - 1` is valid. Perhaps you could use this as a chance to clarify.

Reply:
Done.